Tensile/KernelWriterAssembly.py

```diff
@@ -8105,6 +8248,11 @@ def localReadDoVALU(self, kernel, bufferIdx, iui, epsi, tP):
             localReadCode.addInst("s_mov_b32", sgpr(tmpSgpr), hex(0x3f803f80),"CheckValue1: BF16")   # packed 1s
             localReadCode.addCode(self.assert_eq( dbgVgpr, sgpr(tmpSgpr)))
 
+          # TODO - Check if this works
+          if kernel["ProblemType"]["DataType"].isInt8():
+            localReadCode.addInst("s_mov_b32", sgpr(tmpSgpr), hex(0x01010101),"CheckValue1: INT8")   # packed 1s
+            localReadCode.addCode(self.assert_eq( dbgVgpr, sgpr(tmpSgpr)))
+
```

Comment:
This could be removed once we are sure Int8 will not be supported in VALU version